galpy/potential/FDMDynamicalFrictionForce.py

```diff
@@ -0,0 +1,147 @@
+import astropy.units as u
+import numpy as np
+import scipy.special as sp
+from astropy.constants import c, hbar
+from scipy import interpolate
+from scipy.integrate import quad
+
+from galpy.potential import ChandrasekharDynamicalFrictionForce
```

Comment:
This should fix the test failures (and probably failures when you try to run this code)
```suggestion
from . ChandrasekharDynamicalFrictionForce import ChandrasekharDynamicalFrictionForce
```